### DISCUSSION TITLE
How many GitHub Pages repos are supported with a GitHub Pro subscription?

### DISCUSSION THREAD

**Discussion by ghost** [community/community#48423] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

I'd like to host several static web pages on GitHub, all of which in private repos. I guess the private repo part entail a GitHub Pro, but how many such repos (in other words: different domains) are supported? E.g. can I have 4 different GitHub Page repos corresponding to 4 different (custom) domains?

**Comment by melvincwng** [comment, marked as answer]
Hey @sedeki, I believe that should be possible. 

I have seen several GitHub users hosting multiple static websites on GitHub pages (thus indicating multiple repositories were being used).

As long as your static websites do not exceed the bandwidth limits set by GitHub Pages, you should be good to go.

If you need additional reference on what are the limits for GitHub Pages usage, please refer to this [link](https://docs.github.com/en/pages/getting-started-with-github-pages/about-github-pages).

I hope this clarifies and answers your question 😃 

Take care, and have a good day 👋
